src/main/scala/no/ndla/articleimport/integration/MigrationApiClient.scala

```diff
@@ -96,10 +96,12 @@ case class MigrationMainNodeImport(titles: Seq[MigrationContentTitle],
         if (nType == "emneartikkel") ArticleType.TopicArticle
         else ArticleType.Standard)
 
+    val lic = if (license.exists(_.trim.nonEmpty)) license else None
```

Comment:
Det ligger en funksjon `emptySomeToNone` i package objektet for `model.domain`. Tror den kan brukes her